### PR TITLE
WIP: add internal lineage RFC to open source

### PR DESCRIPTION
**Pull request by feng-tao**
Move the internal google doc RFC to open source, I will refine it or others will help to refine it once we have the UX mock.

Signed-off-by: feng-tao <fengtao04@gmail.com>

### CODE REVIEW

**Comment by feng-tao** [comment]
this RFC is not ready for review

File: active-rfcs/0003-lineage.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [amundsen-io/rfcs#0000](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
```
Comment (Golodhros) [inline active-rfcs/0003-lineage.md:3]:
```suggestion
- RFC PR: [amundsen-io/rfcs#0009](https://github.com/amundsen-io/rfcs/pull/9)
```

File: active-rfcs/0003-lineage.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+## Support Lineage In Amundsen
+
+
+This RFC drafts how lineage should be supported in Amundsen and why the feature is important for the trust in data initiative.
```
Comment (Golodhros) [inline active-rfcs/0003-lineage.md:9]:
```suggestion
This RFC drafts how lineage should be supported in Amundsen and why the feature is important.
```

File: active-rfcs/0003-lineage.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+
+
+*   _Table level lineage_:_ _which table is used as upstream / downstream of the table (dataset).
+*   _Task and table lineage_: which Airflow task produces the given table. Which table is used by which set of tasks.
+*   _Task level lineage_: which Airflow task is upstream / downstream for another Airflow task.
+*   Column lineage: It tracks information in fine detail, at the level of particular columns rather than entire tables. It shows how data of a given column is generated based on upstream columns, and how it affects which downstream columns.
+*   _Service and event lineage_: which service produces and generates the message for a given event table.
+
+Currently Amundsen doesn’t support showing Table level lineage in the UI but instead redirect users to use  , which is a 3rd party tool at Lyft, to refer to the lineage information.
```
Comment (Golodhros) [inline active-rfcs/0003-lineage.md:26]: to use X , 

Something missing here, right?

File: active-rfcs/0003-lineage.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+![](imgs/003/img1.png)
+
+There are several caveats of this approach which negatively impacts user trust towards Lyft data asset:
+
+
+
+*   The link to in-house lineage tool doesn’t always work. If the table/dataset is not indexed in in-house lineage tool, the user will get a 404 (e.g [user report](https://lyft.slack.com/archives/CBHCUQB2Q/p1584997367078000)).
```
Comment (Golodhros) [inline active-rfcs/0003-lineage.md:34]: We'll probably want to remove that report from our internal Slack

File: active-rfcs/0003-lineage.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+*   The UI in in-house lineage tool is hard to browse the actual lineage as it displays the full lineage instead one level at a time (E.g [graph](https://in-house lineage tool.lyft.net/lineage-report/relation/warehouse.core.fact_rides.html?cwRelationContent=graph&cwRelationGraph=dataflow) )
```
Comment (Golodhros) [inline active-rfcs/0003-lineage.md:35]: Another Lyft only link

File: active-rfcs/0003-lineage.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+*   The lineage information provided by in-house lineage tool has lots of noise. For example, it will put s3 path as upstream/downstream for a given data set.
+*   Other popular OSS projects (e.g Apache Atlas) provide the UI support on lineage which is a feature gap for Amundsen adoption in the community.
+
+
+### Why is lineage important for trust in data?
```
Comment (Golodhros) [inline active-rfcs/0003-lineage.md:40]:
```suggestion
### Why is lineage important?
```

**Comment by feng-tao** [comment]
close this pr to favor of https://github.com/amundsen-io/rfcs/pull/24 which is a more updated version.

**Pull request closed without merging**